cosmwasm/packages/osmosis-swap/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod swaprouter;
```

Comment:
Adding the comment here cause I can't comment on swaprouter.rs because it is a move.

I think we shouldn't move all of the contents of swaprouter::msg here. For example, the instantiate msg and queries and query responses aren't used by cross chain swaps or outpost, so we can leave those in `swaprouter`

Reply:
I think the best practice here is to leave all swaprouter::msg in one place, so that we don't have to look at different places.